.NET/Microsoft.Recognizers.Text.DateTime/Utilities/TimexUtility.cs

```diff
@@ -260,9 +260,34 @@ public static DateObject OffsetDateObject(DateObject date, int offset, DatePerio
 
         public static string MergeTimexAlternatives(string timex1, string timex2)
         {
+            if (timex1.Equals(timex2))
```

Comment:
In such comparisons, always specify StringComparison.ordinal as it's not a language-dependant comparison.